yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/block/FileOptions.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * FileOptions.java
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2021-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.apple.foundationdb.relational.yamltests.block;
+
+import com.apple.foundationdb.relational.yamltests.Matchers;
+import com.apple.foundationdb.relational.yamltests.YamlExecutionContext;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Assumptions;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Block that configures aspects of the test that do not require a connection yet.
+ * <p>
+ *     This supports the following sub-commands:
+ *     <ul>
+ *         <li>{@code supported_version}: if this is set, it will disable the test when running against a server
```

Comment:
```suggestion
 *         <li>{@code supported_version}: if this is set, it will enable the test only when running against a server
```

Reply:
I'd prefer to continue to use `disable`, to align with the junit annotations, but reworded it, because it should be disabled when running older versions, not newer versions.